vars/fedoraInfraTox.groovy

```diff
@@ -9,13 +9,14 @@ def call(Closure body) {
                                       args: '${computer.jnlpmac} ${computer.name}',
                                       workingDir: "/workdir")
     ]
-    def active_fedoras = ["f28", "f29", "rawhide"]
+    def active_fedoras = ["f29", "f30", "f31", "rawhide"]
 
     active_fedoras.each { fedora ->
```

Comment:
@bstinsonmhk since f29 doesn't have poetry, do you suggest splitting this?
or some other way to handle this situation?